cobalt/script/v8c/BUILD.gn

```diff
@@ -147,6 +147,7 @@ target(final_executable_type, "v8c") {
     "//cobalt/base",
     "//cobalt/script:script",
     "//cobalt/script:standalone_javascript_runner",
+    "//net:http_server",
```

Comment:
This seems out of place - what pulls it in ? I wouldn't expect http_Server deps in v8c

Reply:
v8c somehow depends on //net which pulls in net:http_server

Reply:
Ill fix this by adding http_server as one of net dependencies